BackendWebApp/src/main/java/uk/ac/ebi/biostd/webapp/server/endpoint/submit/FormSubmitServlet.java

```diff
@@ -135,8 +140,7 @@ protected void service(HttpServletRequest req, HttpServletResponse response, Ses
                 ignPrm != null && ("true".equalsIgnoreCase(ignPrm) || "yes".equalsIgnoreCase(ignPrm) || "on"
                         .equalsIgnoreCase(ignPrm) || "1".equals(ignPrm));
 
-        SubmissionReport res = BackendConfig.getServiceManager().getSubmissionManager()
-                .createSubmission(data, fmt, "UTF-8", act, sess.getUser(), validateOnly, ignAbsFiles);
+        SubmissionReport res = submissionManager.createSubmission(data, fmt, "UTF-8", act, sess.getUser(), validateOnly, ignAbsFiles);

```

Comment:
"UTF-8" can be replaced with `java.nio.charset.StandardCharsets.UTF_8`

Reply:
true, but need to change createSubmission parameter types.. that is another story, but I could try to

Reply:
you are right, my mistake I was looking for

```
org.apache.commons.codec.CharEncoding.UTF_8
``